pom.xml

```diff
@@ -60,7 +60,7 @@
         <maven.gpg.plugin.version>1.4</maven.gpg.plugin.version>
         <maven.surefire.plugin.version>2.18.1</maven.surefire.plugin.version>
         <maven.failsafe.plugin.version>2.14</maven.failsafe.plugin.version>
-        <maven.findbugs.plugin.version>3.0.0</maven.findbugs.plugin.version>
+        <maven.findbugs.plugin.version>3.0.5</maven.findbugs.plugin.version>
```

Comment:
What about upgrading to spotBugs?

Reply:
I'm fine with it - never used it, though. For now, I just converted existing jobs without really changing anything

Reply:
I know that findbugs is not maintained anymore so we change it whenever we have time. This is an example spotBugs usage. https://github.com/hazelcast/hazelcast-azure/blob/master/pom.xml#L32